lib/ens.ts

```diff
@@ -32,21 +16,45 @@ export async function resolveAddressToEnsName(
 ): Promise<ENSResolveResult> {
   invariant(utils.isAddress(address), "address must be a valid address");
 
-  return {
-    ensName: await getMainnetProvider().lookupAddress(address),
-    address,
-  };
+  try {
+    const ensName = await resolveAddress(address);
+
+    return {
+      ensName,
+      address,
+    };
+  } catch (e) {
+    // if the name is not registered, resolveName throws an error
+    // we can ignore this error and return the address
+
+    return {
+      ensName: null,
+      address,
+    };
+  }
 }
 
 export async function resolveEnsNameToAddress(
   ensName: string,
 ): Promise<ENSResolveResult> {
   invariant(isEnsName(ensName), "ensName must be a valid ens name");
 
-  return {
-    ensName,
-    address: await getMainnetProvider().resolveName(ensName),
-  };
+  try {
+    const address = await resolveEnsSdk(ensName);
```

Comment:
which clientId / secret key does this use?

Reply:
whatever we use for the rest of the dashboard, anyway we can't do this right now because resolveEns in the sdk is fake and doesn't actually resolveEns, both resolveEns and resolveAddress returns the address from the ens 🙃